topics/statistics/tutorials/RNN/tutorial.md

```diff
@@ -0,0 +1,374 @@
+  and anything less than 0.5 a negative one. The model config can be downloaded as a JSON file.
+{: .comment}
+
+### **Create a deep learning model**
+
+> ### {% icon hands_on %} Hands-on: Model builder (Optimizer, loss function, and fit parameters)
+>
+> 1. **Create deep learning model** with an optimizer, loss function and fit parameters {% icon tool %} with the following parameters:
+>    - *"Choose a building mode"*: `Build a training model`
+>    - *"Select the dataset containing model configuration"*: Select the *Keras Model Config* from the previous step.
+>    - *"Do classification or regression?"*: `KerasGClassifier`
+>    - In *"Compile Parameters"*:
+>        - *"Select a loss function"*: `binary_crossentropy` 
+>        - *"Select an optimizer"*: `Adam - Adam optimizer `
+>        - *"Select metrics"*: `acc/accuracy`
+>    - In *"Fit Parameters"*:
+>        - *"epochs"*: `2`
+>        - *"batch_size"*: `128`
+>    - Click *"Execute"*
+{: .hands_on}
+
+> ### {% icon comment %} Comment
+> A loss function measures how different the predicted output is versus the expected output. For binary classification problems, we use 
+  *binary cross entropy* as loss function. Epochs is the number of times the whole training data is used to train the model. Setting *epochs* to 2 
+  means each training example in our dataset is used twice to train our model. If we update network weights/biases after all the training data is 
+  feed to the network, the training will be very slow (as we have 25000 training examples in our dataset). To speed up the training, we present 
+  only a subset of the training examples to the network, after which we update the weights/biases. *batch_size* decides the size of this subset. 
+  The model builder can be downloaded as a zip file.
+{: .comment}
+
+### **Deep learning training and evaluation**
+
+> ### {% icon hands_on %} Hands-on: Training the model
+>
+> 1. **Deep learning training and evaluation** conduct deep training and evaluation either implicitly or explicitly {% icon tool %} with the following parameters:
+>    - *"Select a scheme"*: `Train and Validate`
+>    - *"Choose the dataset containing pipeline/estimator object"*: Select the *Keras Model Builder* from the previous step. 
+>    - *"Select input type:"*: `tabular data`
+>        - *"Training samples dataset"*: Select `X_train` dataset
+>        - *"Choose how to select data by column:"*: `All columns`
+>        - *"Dataset containing class labels or target values"*: Select `y_train` dataset
+>        - *"Choose how to select data by column:"*: `All columns`
+>    - Click *"Execute"*
+>
+>
+{: .hands_on}
+
+> ### {% icon comment %} Comment
+> The training step generates 3 datasets. 1) accuracy of the trained model, 2) the trained model, downloadable as a zip file, and 3) the trained 
+  model weights, downloadable as an hdf5 file. These files are needed for prediction in the next step. 
+{: .comment}
```

Comment:
All of these post-tool comment boxes probably don't need to be a comment box.

```suggestion
The training step generates 3 datasets. 1) accuracy of the trained model, 2) the trained model, downloadable as a zip file, and 3) the trained 
  model weights, downloadable as an hdf5 file. These files are needed for prediction in the next step. 
```

Reply:
Moved them all out of comment sections.